src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.Versions.props

```diff
@@ -15,6 +15,7 @@
     <AndroidSdkEmulatorVersion Condition="'$(AndroidSdkEmulatorVersion)' == ''"></AndroidSdkEmulatorVersion>
     <AndroidSdkPlatformVersion Condition="'$(AndroidSdkPlatformVersion)' == ''">android-34</AndroidSdkPlatformVersion>
     <AndroidNdkVersion Condition="'$(AndroidNdkVersion)' == ''">26.1.10909125</AndroidNdkVersion>
+    <JavaSdkVersion Condition="'$(JavaSdkVersion)' == ''">17.0.8.1</JavaSdkVersion>
```

Comment:
I think `$(JdkVersion)` was also a candidate name for this property? `$(JavaSdkVersion)` may be a better fit, as it more closely matches the `$(JavaSdkDirectory)` property that can be used to specify the path to the JDK.

Reply:
I think this is the only doc that mentions `InstallAndroidDependencies`:

https://github.com/dotnet/docs-maui/blob/main/docs/get-started/installation.md

I didn't make a public property for the JDK version here: https://github.com/xamarin/monodroid/commit/dbc9c67e7bd45744e5174dd7e94e5b642418bef9